src/components/RadioButton/RadioButton.st.css

```
@@ -65,6 +65,12 @@
     cursor: pointer;
 }
 
+.wrapperLabelOnly{
```

So this can actually be:
```suggestion
.wrapper:not(.withChildren) {
```
right?